Source/w_wad.c

```diff
@@ -364,6 +364,40 @@ int (W_CheckNumForName)(register const char *name, register int name_space) // [
   return i;
 }
 
+// W_FindNumFromName, an iterative version of W_CheckNumForName
```

Comment:
We already have simpler code to load multiple DEHACKED lumps, maybe this could get adjusted accordingly: 
https://github.com/fabiangreffrath/woof/blob/49253820e3e28c8684194317e48b7b6d40874ef1/Source/d_main.c#L1572